CMakeLists.txt

```diff
@@ -7,9 +6,22 @@ ENDIF()
 
 IF(NOT KOKKOS_HAS_TRILINOS)
   CMAKE_MINIMUM_REQUIRED(VERSION 2.8.11 FATAL_ERROR)
+  # hack in openmp
+  IF (Kokkos_ENABLE_OpenMP)
+    FIND_PACKAGE (OpenMP REQUIRED)
+    IF (OPENMP_FOUND)
+      SET (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} ${OpenMP_CXX_FLAGS}")
+      SET (CMAKE_Fortran_FLAGS "${CMAKE_Fortran_FLAGS} ${OpenMP_C_FLAGS}")
+    ENDIF ()
+  ENDIF (Kokkos_ENABLE_OpenMP)
+  IF (Kokkos_ENABLE_CUDA)
+    SET (CMAKE_CXX_COMPILER "${CMAKE_SOURCE_DIR}/bin/nvcc_wrapper")
+  ENDIF ()
   INCLUDE(cmake/tribits.cmake)
 ENDIF()
 
+SET (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Comment:
I am meaning more something like have a toolchain file called cpp11toolchain.cmake, like this:

```cmake
set (CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
```

Then build with cmake like this:

```
cmake -DCMAKE_TOOLCHAIN_FILE=cpp11toolchain.cmake ..
```

Or use an alias to save typing like this:

```
alias cmake11=cmake -DCMAKE_TOOLCHAIN_FILE=cpp11toolchain.cmake
cmake11 ..
```